README.md

```diff
@@ -100,7 +111,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+An outline of the attributes of an entity. A one-to-many relationship is an example such as a customer can make many orders. This is represented in a SQL database through a customer being able to make multiple orders but orders only belonging to one customer.
```

Comment:
It's represented in the database with the Orders table having a `customer_id` column. This is how an Order "belongs to" a Customer.